iaso/models/org_unit.py

```diff
@@ -625,7 +625,7 @@ class Statuses(models.TextChoices):
         REJECTED = "rejected", _("Rejected")
         APPROVED = "approved", _("Approved")
 
-    uuid = models.UUIDField(default=uuid.uuid4, editable=False)
+    uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
```

Comment:
Does this mean that sending a new Change Request with the same UUID will produce a 400 or that it will update it?

Reply:
In fact, sending a new Change Request with an existing UUID will trigger an `IntegrityError` in the serializer, resulting in a 500 HTTP response.

This will help us to detect the duplicate error source.

If you need something else, just tell me and I'll handle it in the code.

Reply:
To be confirmed with @madewulf, but we usually don't want synchronization from device to backend to be throwing errors (not to block people on the field). So I'm wondering if that could be an issue